lib/tasks/ssh_exec.rake

```diff
@@ -2,19 +2,18 @@ namespace :ssh_exec do
   desc "Test interaction with SSH"
   task test: :environment do
     s = Server.first
+    output = ""
     SshExecution.new(s).execute_with_block do |ssh|
       ssh.open_channel do |ch|
-        ch.exec "dokku apps:destroy test3" do |ch, success|
+        ch.exec "dokku postgres:export formbox_postgres" do |ch, success|
```

Comment:
This is my test file to play with ssh tasks, I think I should add this to git ignore xD